perception/image_projection_based_fusion/launch/roi_cluster_fusion.launch.xml

```diff
@@ -18,23 +17,15 @@
   <arg name="input/camera_info7" default="/camera_info7"/>
   <arg name="input/clusters" default="clusters"/>
   <arg name="output/clusters" default="labeled_clusters"/>
+  <arg name="param_path" default="$(find-pkg-share image_projection_based_fusion)/config/roi_cluster_fusion.param.yaml"/>
   <arg name="sync_param_path" default="$(find-pkg-share image_projection_based_fusion)/config/roi_sync.param.yaml"/>
-  <arg name="remove_unknown" default="true"/>
-  <arg name="trust_distance" default="100.0"/>
 
   <!-- for eval variable-->
   <arg name="input_rois_number" default="$(var input/rois_number)"/>
 
   <!-- debug -->
   <!-- cspell: ignore minx, maxx, miny, maxy, minz, maxz  -->
   <arg name="debug_mode" default="false"/>
```

Comment:
@ktro2828 shall we also move this to (roi_sync).param.yaml too?

Reply:
I removed all `debug_mode` variables in launcher and moved into `roi_sync.params.yaml` in https://github.com/autowarefoundation/autoware.universe/pull/4902/commits/16faca87b7bec0b3be8ef8f18a6103d06ed97f0b